docs/installation/requirements.md

```diff
@@ -87,22 +87,23 @@ Ensure that the time setting in the BIOS of your server is set to the current ti
 
 ## 🖥️ Supported guest OS
 
-Officially supported OS. But there's a lot more than can run on XCP-ng.
+We only officially support operating systems which still receive support from their publisher. But there's a lot more than can run on XCP-ng.
 
 ### Windows based
 
-* Windows Server 2012, 2016, 2019, 2022
```

Comment:
But those versions are rather different, so we might tell "all versions from 2012" IMHO.

Reply:
2012 is not supported anymore. I'll add the list back but without 2012.

Reply:
I also added a word in the 'info' block.